packages/xsnap/lib/console-shim.js

```diff
@@ -1,16 +1,13 @@
 /* global globalThis */
+
+// We use setQuote() below to break the cycle
+// where SES requires console and console is
+// implemented using assert.quote from SES.
+let quote = v => (typeof v === 'string' ? v : '[?]');
```

Comment:
With the guard below, this can probably reduce to `v => '[?]'`.

Reply:
That's the way it was originally, but during debugging everything was coming out as `[?]`, which was annoying.

I suppose now that those bugs are fixed, this can be simplified.

Reply:
Don’t trouble for it! Debugging without a call to setQuote is a good reason to leave it as-is. I have been thinking a lot lately about how code lies on the spectrum from flexible to strong.

Reply:
too late. :)